### ISSUE TITLE
MAJA seems mandatory during installation

### ISSUE THREAD

**Issue by savmickael** [Sen4CAP/Sen4CAP#1]
Even if the system can run with Sen2cor processor, the MAJA processor is mandatory during installation. It is possible to have the possibility to skip this point during the installation ?

**Comment by lnicola** [comment]
Yes, we should allow installations without MAJA if that's not already possible. We'll look into it soon.
